android/src/main/res/values/styles.xml

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <style name="Theme.EdgeToEdge.Common" parent="Theme.AppCompat.DayNight.NoActionBar">
-        <item name="android:fitsSystemWindows">false</item>
-        <item name="android:windowDrawsSystemBarBackgrounds">true</item>
+    <style name="Theme.EdgeToEdge.Base" parent="Theme.AppCompat.DayNight.NoActionBar">
+        <item name="android:windowLightStatusBar">true</item>
     </style>
 
-    <style name="Theme.EdgeToEdge.DayNight" parent="Theme.EdgeToEdge.Common">
-        <item name="darkContentBarsStyle">true</item>
+    <style name="Theme.EdgeToEdge.Common" parent="Theme.EdgeToEdge.Base">
+        <item name="android:fitsSystemWindows">false</item>
+        <item name="android:windowDrawsSystemBarBackgrounds">true</item>
+        <item name="android:statusBarColor">@android:color/transparent</item>
     </style>
 
-    <style name="Theme.EdgeToEdge" parent="Theme.EdgeToEdge.DayNight">
-        <item name="android:windowTranslucentStatus">true</item>
-        <item name="android:windowTranslucentNavigation">true</item>
+    <style name="Theme.EdgeToEdge" parent="Theme.EdgeToEdge.Common">
+        <item name="android:navigationBarColor">@color/systemBarDarkScrim</item>
```

Comment:
I would like to suggest adding a variant with `Theme.AppCompat.Light.NoActionBar` theme, as not all applications that have to be created are designed to support dark theme and dynamic switching.

Reply:
Something like this

`Theme.EdgeToEdge` - default (current implementation)
`Theme.EdgeToEdge.Light` - will be using `Theme.AppCompat.Light.NoActionBar` theme under the hood